pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -2117,19 +2117,13 @@ func handleClusterUpdateStatusImpl(ctxArg interface{}, key string,
 		log.Errorf("handleClusterUpdateStatusImpl invalid type in configArg: %v", statusArg)
 		return
 	}
-	log.Noticef("handleClusterUpdateStatusImpl key:%s obj:%v", key, req)
-
 	state := getDeviceState(ctx)
-	//if state != DEVICE_STATE_BASEOS_UPDATING {
-	// Ignore if not in baseos updating state
-	//	return
-	//}
-	log.Noticef("handleClusterUpdateStatusImpl devicestate:%s", state)
+	log.Noticef("handleClusterUpdateStatusImpl key:%s obj:%v devicestate:%s", key, req, state)
 	triggerPublishKubeClusterUpdateStatus(ctx)
 }
 
 func triggerPublishKubeClusterUpdateStatusToDest(ctxPtr *zedagentContext, dest destinationBitset) {
-	log.Notice("Triggered PublishKubeClusterUpdateStatus")
+	log.Function("Triggered PublishKubeClusterUpdateStatus")
```

Comment:
Ins't it supposed to be log.Functionf

Reply:
The only argument here is the string so Function should be ok

Reply:
ok